src/ploneorg/addonlisting/vocabularies.py

```diff
@@ -42,21 +43,32 @@ def addon_type_vocabulary(context):
 @provider(IVocabularyFactory)
 def addon_pypi_state_vocabulary(context):
     return SimpleVocabulary([
-        SimpleTerm(value=u'status-1', title=u'Development Status :: 1 - Planning'),
```

Comment:
my opinion: it's more legible and less error-prone to have these terms on just one line, even if it violates a hard code formatting standard